Annotation/alternative_golden_set.json

```diff
@@ -946,7 +958,7 @@
                     }
                 ],
                 "type": "alternative",
-                "clause": "",
+                "clause": "whether any given piece of your writing is carefully polished already or just a new idea you dashed off on the train today",
                 "embedded clauses": []

```

Comment:
coordinated EC with "that a critique might be helpful"

Reply:
I don't see it as an EC, but as a degree clause: developed well enough that a critique might be helpful. Another example of this type of clause would be: I like the book so much that I'm going to buy it